### DISCUSSION TITLE
Webradios packages

### DISCUSSION THREAD

**Discussion by Rasb74** [rern/rAudio#196] [Feature Requests]
Simple idea, is there any possibilities to propose some web radios packages, including the pls files and cover arts accordingly ?
Would be easier for import process

**Comment by akelge** [comment]
I was looking at it, so far I sent a patch to improve importing of .pls files, but I think that packages would be a good thing, especially making easy the management of cover art.
I can't promise, but I can give it a try, as I am directly interested for personal purposes ;)

**Comment by akelge** [comment]
> Simple idea, is there any possibilities to propose some web radios packages, including the pls files and cover arts accordingly ?
> Would be easier for import process

BTW, are you aware of any standard format for Web radio packages? I jotted down a prototype using my own format, but if there is any standard I would stick with it

**Reply by Rasb74** [reply]
Hi, I don't have specific idea of the package standard. As user post of view, I think it will be important to separate Flac and MP3 flow folders.

**Reply by akelge** [reply]
Are there web radios that transmit in Flac format? Well, ok, I will keep this in account

**Reply by R101G** [reply]
I am not sure what you are doing, but don't forget Opus and AAC too. If you are going to treat each streaming format differently, it will get very complicated.

**Reply by akelge** [reply]
I am not sure myself anymore :) Do you have some sample links of webradio serving more than one format?

**Reply by R101G** [reply]
The default Webradio entries are flac. The other RP feeds are here https://radioparadise.com/listen/stream-links.

It seems easier to to copy and paste a URL and download the station logo, rather than import radio playlists.
